docs/Glossary.md

```diff
@@ -0,0 +1,20 @@
+---
+id: Glossary
+title: Glossary of Arbitrum Terms
+custom_edit_url: https://github.com/OffchainLabs/arbitrum/edit/master/docs/Glossary.md
+---
+
+* Assertion: A claim, made by a validator, about what the contracts on a chain will do. An assertion is considered _pending_ until it is confirmed.
+* Arbitrum Chain: A chain running on Arbitrum, containing some contracts. Many Arbitrum chains can exist at the same time.
+* Arbitrum executable file: The output of the Arbitrum compiler, usually in a file with the .ao file extension. An Arbitrum executable has all of the information needed to launch an Arbitrum Chain.
+* AVM: The Arbitrum Virtual Machine
+* Challenge: When two stakers disagree about the correct verdict on an assertion, those stakers can be put in a challenge. The challenge is refereed by the EthBridge. Eventually one staker wins the challenge. The loser forfeits their stake. Half of the loser's stake is given to the winner, and the other half is burned. 
+* Client: A program running on a user's machine, often in the user's browser, that interacts with contracts on an Arbitrum chain and provides a user interface.
+* Compiler: A part of the Arbitrum toolchain that takes a set of Solidity contracts and compiles them into an Arbitrum executable file.
+* Confirmation: The final decision by an Arbitrum chain to accept a node as being a settled part of the chain's history. When a node is confirmed, any funds paid out by the chain to the main Ethereum chain are transferred.
+* EthBridge: A group of contracts running on the Ethereum chain, which act as a record-keeper and rule enforcer for Arbitrum chains.
+* Node: A particular point in the history of an Arbitrum Chain. A node corresponds to a sequence of assertions that have been made, and a verdict about whether each of those assertions was accepted.
```

Comment:
The term node is potentially confusing because of the naming collision between this and the usage to describe blockchain client software ex. "Ethereum Node". We should consider whether there is a way to disambiguate this.

Reply:
Agree that "node" is not ideal.  Logically, it's something like a proposed history of the chain.  "Chain state" isn't quite right, because the state of the chain includes a bunch of these nodes.  

In one version of the original spec, I called the boxes "verdicts".  The idea being that it represents a judgment about which assertions have been accpted.

Thoughts?

Reply:
I'm going to go with "chain state".